backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -479,7 +440,88 @@ private static void addCorrectionNote(
     }
   }
 
-  private static void addValue(String resultCode, Observation observation, String resultDisplay) {
+  public static Set<Observation> convertToAOEObservations(
+      String eventId, AskOnEntrySurvey surveyData) {
+    var observations = new HashSet<Observation>();
+    var symptomaticCode =
+        createLoincConcept(
+            "95419-8",
+            "Has symptoms related to condition of interest",
+            "Has symptoms related to condition of interest");
+    if (surveyData.getNoSymptoms()) {
+      // user reported as not symptomatic
+      observations.add(
+          createAOEObservation(
+              eventId + "-symptomatic", symptomaticCode, createYesNoUnkConcept(false)));
+    } else if (surveyData.getSymptoms().containsValue(Boolean.TRUE)) {
+      // user reported as symptomatic
+      observations.add(
+          createAOEObservation(
+              eventId + "-symptomatic", symptomaticCode, createYesNoUnkConcept(true)));
+      if (surveyData.getSymptomOnsetDate() != null) {
+        observations.add(
+            createAOEObservation(
+                eventId + "-onset",
+                createLoincConcept(
+                    "11368-8",
+                    "Illness or injury onset date and time",
+                    "Illness or injury onset date and time"),
+                new DateTimeType(surveyData.getSymptomOnsetDate().toString())));
+      }
+    } else {
+      // if neither no symptoms nor any symptoms checked, AoE form was not completed
+      observations.add(
+          createAOEObservation(
+              eventId + "-symptomatic", symptomaticCode, createYesNoUnkConcept(null)));
+    }
+    return observations;
+  }
+
+  public static Observation createAOEObservation(
+      String uniqueName, CodeableConcept code, Type value) {
+    var observation = new Observation();
+    observation.setId(UUID.nameUUIDFromBytes(uniqueName.getBytes()).toString());
+    observation.setStatus(ObservationStatus.FINAL);
+
+    var identifier = new Identifier();
+    identifier.setUse(Identifier.IdentifierUse.OFFICIAL);
+    identifier.setType(
+        createLoincConcept("81959-9", "Public health laboratory ask at order entry panel", null));
+    observation.setIdentifier(List.of(identifier));
+
+    observation.setCode(code);
+    observation.setValue(value);
+    return observation;
```

Comment:
The prefered way with the FHIR library is actually to chain everything together so this could look like: 
```java
    var observation =
        new Observation().setStatus(ObservationStatus.FINAL).setCode(code).setValue(value);
    observation.setId(UUID.nameUUIDFromBytes(uniqueName.getBytes()).toString());
    observation
        .addIdentifier()
        .setUse(Identifier.IdentifierUse.OFFICIAL)
        .setType(
            createLoincConcept(
                "81959-9", "Public health laboratory ask at order entry panel", null));
```